app/controllers/short_url_requests_controller.rb

```diff
@@ -8,6 +8,7 @@ def index
   end
 
   def show
+    @existing_redirect = Redirect.where(from_path: @short_url_request.from_path).first
```

Comment:
The ticket describes a scenario where multiple people have made multiple requests (plus an existing redirect already exists -- a previous request for a short url was accepted). It turns out that this scenario is very rare and the main (initial) piece of work is aimed at stopping reviewers overwriting existing redirects.

This piece of work solves that user need, with subsequent pieces of work (the next two short URL manager stories) surfacing subsequent requests to the requestor.

I should have updated the story to reflect that, I apologise.